canonical/rebac-admin
# WD-12400 - feat: add user roles tab

## Done

- Added roles tab to user details.

## QA

- Go to a user's detail page and click on the roles tab.
- Check that you can add roles and that they get displayed in the table.

## Details

https://warthogs.atlassian.net/browse/WD-12400
